lib/const.rb

```diff
@@ -194,11 +197,11 @@
 OPT
 
 # Use ninja or samurai
-if ENV['CREW_NINJA'].to_s.downcase == 'ninja'
-  CREW_NINJA = 'ninja'
-else
-  CREW_NINJA = 'samu'
-end
+CREW_NINJA = if ENV['CREW_NINJA'].to_s.downcase == 'ninja'
+               'ninja'
+             else
+               'samu'
+             end
```

Comment:
Immediate if instead?  `CREW_NINJA = ENV['CREW_NINJA'].to_s.downcase == 'ninja' ? 'ninja' : 'samu'`

Reply:
That code isn't very readable though